src/inspect.js

```diff
@@ -23,7 +25,7 @@ function inspectPackageJSON(info) {
   } catch (error) {
     let log = logger('migrate:inspect:js');
 
-    if (error.code === 'MODULE_NOT_FOUND') {
+    if (error.message === 'Missing package.json') {
       log.warn('Could not find package.json in current directory');
     } else {
       log.error('Encountered an error inspecting package.json');
```

Comment:
Then this code can be refactored to not need this if/else at all. Just logging all errors as it would normally do.